README.md

```diff
@@ -426,10 +426,17 @@ The former runs scripted configuration steps that do not change the state of the
 On unexpected failures the command will try to perform a rollback when possible to ensure that the environment continues to function.
 
 ```bash
-./bin/ck8s upgrade vX.Y prepare
-./bin/ck8s upgrade vX.Y apply
+./bin/ck8s upgrade both vX.Y prepare
+./bin/ck8s upgrade both vX.Y apply
```

Comment:
Hmm, I feel like this could be a nice change as to keep it backwards compatible, but I also feel like the commands should follow the same standard (to some degree) which currently would be taking `<wc|sc>` (and `both` in this case) as an argument, but I am conflicted. If anyone else has a preference on this let me know

Reply:
I agree with Anders. It make the cli feel more standardised and the upgrade per cluster more of a feature rather than hacky exception.

Reply:
Sorry, this exposes my time away from the project. :smile: I completely forgot that the cluster name was a common argument in the CLI for other commands.

Nice work!